pegasus/src/social_metadata.rb

```diff
@@ -0,0 +1,155 @@
+# Centralized social metadata for a few key pages:
+#   code.org/
+#   csedweek.org/
+#   hourofcode.com/
+#   code.org/challenge
+
+def get_social_metadata_for_page(request)
+  # Important:
+  #   - image should always come before video
+  #   - description should always come before description_twitter
+  social_tags = {
+    "code.org" => {
+      "soon-hoc" => {
+        title: I18n.t(:og_title),
+        description: I18n.t(:og_description),
+        image: {path: "/images/fit-1220/social-media/default-og-image.jpg", width: 1220, height: 640}
+      },
```

Comment:
Do the hard coded image sizes work responsively?

Reply:
These were the existing sizes used, and are just provided as social tags, not used in the pages themselves.